website/api/models/User.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
 
     organization: {
       type: 'string',
-      required: true,
+      required: false,
```

Comment:
`required:false` is always the default for model attributes, so it can be omitted.

Reply:
I've removed that line